### ISSUE TITLE
Questions about equations in the paper.

### ISSUE THREAD

**Issue by frankkim1108** [liuff19/ReconX#4]
Hello, @liuff19 

Recently, I found your project very interesting and started reading your paper. However, I have some questions on some equations in the paper.

1. 
For equations (12)
$$\ L_{\text{diffusion}} = \mathbb{E}{x \sim p, \epsilon \sim \mathcal{N}(0, I), c{\text{view}}, c_{\text{struc}}, t} \left[ \|\epsilon - \epsilon_{\theta} (x_t, t, c_{\text{view}}, c_{\text{struc}})\|^2_2 \right] $$

in the paper it says that $x_t$ is the noise latent from the ground-truth views of the training data. Which ground-truth view are you referencing from?

2.
In section 5.4 it says that
'For the generated frames $\\{I_i\\}_{i=1}^{K{\prime}}$ we denote $\hat{C}_i$ and $C_i$ the per-pixel color value for generated and ground-truth view $i$.'

What do you mean by ground-truth view $C_i$?

It also appears in equation (13)

$$ L_{I_i} = - \log \left( \frac{1}{\sqrt{2 \pi \sigma_i^2}} \exp \left( -\frac{\|C{\prime}_i - C_i\|^2}{2 \sigma_i^2} \right) \right) $$

3.
For equation (14)
$$\  L_{\text{conf}} = \sum_{i=1}^{K{\prime}} C_i \left( \lambda_{\text{rgb}} L_1(\hat{I_i}, I_i) + \lambda{\text{ssim}} L_{\text{ssim}}(\hat{I_i}, I_i) + \lambda{\text{lpips}} L_{\text{lpips}}(\hat{I_i}, I_i) \right) $$

in this equation it seems like the loss is calculated between the 32 generated frames $\hat{I_i}$ with its GT frames $I_i$.
Which GT frames are you comparing with? Is it the input sparse view? or is it from the train dataset video?

Thank you in advance for your time to reply to this issue.

Best regards 
Frank

**Comment by wzy-99** [comment]
Hello, @liuff19

Can you tell more detail about the process of getting ![image](https://github.com/user-attachments/assets/53882290-dd9d-400f-9974-b506c0deb358)

As the paper:

![image](https://github.com/user-attachments/assets/03ca6ebe-8d88-4358-a161-a38166e4cef9)

Does the equation just get only one  ![image](https://github.com/user-attachments/assets/53882290-dd9d-400f-9974-b506c0deb358) for training equation (13) for view i? With only one sample, it is able to optimaze equation (13)?

Or may be the equation return a set of  ![image](https://github.com/user-attachments/assets/53882290-dd9d-400f-9974-b506c0deb358) for view i, as:
![image](https://github.com/user-attachments/assets/025aee6b-69e6-4e9d-9680-52f5a3138b7e)


Then we can use the set of data to train equation (13) to get 
![image](https://github.com/user-attachments/assets/77d51a81-a3bc-4287-8f44-b0dbf8b0ea5f) and 
![image](https://github.com/user-attachments/assets/41cec98b-fd33-46fb-81e8-39df8165632a).
